docs/docusaurus.config.js

```diff
@@ -10,6 +10,13 @@ module.exports = {
   plugins: ['@docusaurus/plugin-google-analytics'],
   scripts: ['https://www.chartjs.org/dist/VERSION/Chart.min.js'],
   themeConfig: {
+	algolia: {
```

Comment:
The indentation seems off

Reply:
It should be good as https://github.com/s-pace/Chart.js/blob/patch-1/docs/docusaurus.config.js

Reply:
It's because https://github.com/s-pace/Chart.js/blob/patch-1/docs/docusaurus.config.js has a mix of tabs and spaces. Do you mind converting to all spaces?

Reply:
Sure